frontend/src/scenes/pipeline/transformationsLogic.tsx

```diff
@@ -0,0 +1,199 @@
+                    // Update order if enabling to be at the end of current enabled plugins
+                    // See comment in savePluginConfigsOrder about races
+                    let order = {}
+                    if (enabled) {
+                        const maxOrder = Math.max(
+                            ...Object.values(values.enabledPluginConfigs).map((pc) => pc.order),
+                            0
+                        )
+                        order = { order: maxOrder + 1 }
+                    }
+                    const response = await api.update(`api/plugin_config/${id}`, {
+                        enabled,
+                        ...order,
+                    })
+                    return { ...pluginConfigs, [id]: response }
+                },
+            },
+        ],
+    })),
+    selectors({
+        loading: [
+            (s) => [s.pluginsLoading, s.pluginConfigsLoading],
+            (pluginsLoading, pluginConfigsLoading) => pluginsLoading || pluginConfigsLoading,
+        ],
+        enabledPluginConfigs: [
+            (s) => [s.pluginConfigs],
+            (pluginConfigs) => {
+                return Object.values(pluginConfigs).filter((pc) => pc.enabled)
+            },
+        ],
+        sortedEnabledPluginConfigs: [
+            (s) => [s.pluginConfigs, s.enabledPluginConfigs, s.temporaryOrder],
+            (pluginConfigs, enabledPluginConfigs, temporaryOrder) => {
+                if (!temporaryOrder || Object.keys(temporaryOrder).length === 0) {
+                    return enabledPluginConfigs.sort((a, b) => a.order - b.order)
+                }
+                // If temp order is set return the pluginConfigs in that order
+                const result = Object.entries(temporaryOrder)
+                    .sort(([, aIdx], [, bIdx]) => aIdx - bIdx)
+                    .map(([pluginId]) => pluginConfigs[Number(pluginId)])
+                return result
+            },
+        ],
+        disabledPluginConfigs: [
+            (s) => [s.pluginConfigs],
+            (pluginConfigs) => Object.values(pluginConfigs).filter((pc) => !pc.enabled),
+        ],
+        // This is currently an organization level setting but might in the future be user level
+        // it's better to add the permission checks everywhere now
+        canConfigurePlugins: [(s) => [s.user], (user) => canConfigurePlugins(user?.organization)],
+    }),
+    reducers({
+        reorderModalOpen: [
+            false,
+            {
+                openReorderModal: () => true,
+                closeReorderModal: () => false,
+            },
+        ],
+    }),
+    listeners(({ actions }) => ({
+        savePluginConfigsOrderSuccess: () => {
+            actions.closeReorderModal()
+        },
+    })),
+    afterMount(({ actions }) => {
+        actions.loadPlugins()
+        actions.loadPluginConfigs()
+    }),
+])
+
+const PAGINATION_DEFAULT_MAX_PAGES = 10
+async function loadPaginatedResults(
```

Comment:
Ideally we'd render the first results already when they arrive & display a loading indicator that more results are coming.

Reply:
How would I do that (is there another page doing this)?
On another note there aren't too many results here, so it's likely not a problem.